xml/ay_bigfile.xml

```diff
@@ -14429,6 +14420,22 @@ echo -n 100
        <row>
         <entry>
          <para>
+          <literal>autoyast2</literal>
+         </para>
+        </entry>
+        <entry>
+         <para>
+          Location of the control file for automatic installation,
+          see <xref linkend="Commandline.ay"/> for details. Similar to
```

Comment:
The reference is wrong. https://en.opensuse.org/SDB:Linuxrc#p_autoyast2 states that it is Linuxrc parsing the URL and it uses different schemes than (Auto)YaST.